public/app/features/provisioning/Job/JobContent.tsx

```diff
@@ -29,6 +29,15 @@ export function JobContent({ job, isFinishedJob = false }: JobContentProps) {
             title={t('provisioning.job-status.status.title-job-completed-successfully', 'Job completed successfully')}
           />
         );
+      case 'warning':
+        return (
+          <Alert
+            severity="warning"
+            title={t('provisioning.job-status.status.title-warning-running-job', 'Warning running job')}
+          >
+            {message ?? errors?.join('\n')}
```

Comment:
```suggestion
              {errors?.length ? errors?.join('\n') : message}
```
I think there's more value in showing the actual errors than the message (same for the error state).

Reply:
<img width="1221" height="359" alt="image" src="https://github.com/user-attachments/assets/ff9047f0-3039-45c0-a57c-8c5b9ab2957e" />


The concatenation doesn't seem to be working too well but I will leave it to you to create a better component / style (e.g. bullets) for job errors across the UI. It's not just a problem here.

Reply:
Ticket: https://github.com/grafana/git-ui-sync-project/issues/365 @Clarity-89

Reply:
That's weird, those errors should be shown correctly here. I'll take a look!